core/controllers/moderator_test.py

```diff
@@ -54,17 +55,20 @@ class FeaturedActivitiesHandlerTest(test_utils.GenericTestBase):
 
     EXP_ID_1 = 'exp_id_1'
     EXP_ID_2 = 'exp_id_2'
-    ALBERT_ID = 'albert'
+    user = 'albert'
```

Comment:
"user" seems to mean something different than it usually does... it's a string and not a UserActionsInfo object.